_posts/2016/2016-12-15-hr-ui.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "Working as a UI-Designer at ePages"
+date: "2016-12-15 12:11:00"
+image: blog-header/ui-designer.jpg
+categories: working-at-epages
+authors: ["Rose, Serena", "Verena"]
+---
+
+So far, we have interviewed a [Ruby Developer](https://developer.epages.com/blog/2016/02/04/hr-ruby.html), a [Java Developer](https://developer.epages.com/blog/2016/03/01/hr-java.html), a [QA engineer](https://developer.epages.com/blog/2016/04/12/hr-quality-assurance.html) and a [Software Developer](https://developer.epages.com/blog/2016/06/01/hr-backend.html) to learn about working at ePages.
```

Comment:
@bbader in the last sentence we would have two times "insight".
maybe:
To give you an introduction into our world of design, we've talked to one of our UI Designers, who shared some insights about her daily work.